site-content/process-native-protocol-specs-in-docker.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# Variables
+GO_VERSION="1.20.6"
+GO_TAR="go${GO_VERSION}.linux-amd64.tar.gz"
+GITHUB_REPO_PARSER="https://github.com/martin-sucha/cqlprotodoc.git"  # Replace with the actual parser repo URL
```

Comment:
I'm hesitant to leave this pointing to a living branch.  Pinning it to a SHA would be safest.

And am curious if we could upstream it to here… (but this can happen separately…)

Reply:
Yes, it is better to copy it under `github.com/apache` (I'll leave up to you to which repo). I'm willing to donate the cqlprotodoc code to the Apache software foundation and change the licence to Apache license, Version 2.0.

Reply:
Thanks @martin-sucha.  That's all the approval/consent we need.

@tengu-alt , are you willing (and got the time) to include the contents of the repo in the folder `cqlprotodoc` and adjust accordingly, in this PR ? (don't include its LICENSE file, add to its readme a sentence about where it comes from)

Reply:
> Thanks @martin-sucha. That's all the approval/consent we need.
> 
> @tengu-alt , are you willing (and got the time) to include the contents of the repo in the folder `cqlprotodoc` and adjust accordingly, in this PR ? (don't include its LICENSE file, add to its readme a sentence about where it comes from)

Sure! I will do it.